akka-docs/src/main/paradox/stream/operators/asJavaStream.md

```diff
@@ -0,0 +1,26 @@
+# asJavaStream
```

Comment:
This is another great example for adding the "where" in url and title -- right away from those one would be made aware about existence of StreamConverters; `stream/overview/StreamConverters/asJavaStream.md` would be very nice here